frontend/packages/knative-plugin/src/components/add/event-sources/EventSourcesSelector.tsx

```diff
@@ -29,9 +34,9 @@ const EventSourcesSelector: React.FC<EventSourcesSelectorProps> = ({ eventSource
     <FormSection title="Type" fullWidth>
       <ItemSelectorField
         itemList={eventSourceList}
-        loadingItems={!eventSourceItems}
         name="type"
         onSelect={handleItemChange}
+        loadingItems={!eventSourceItems}
```

Comment:
any reason for changing prop order?

Reply:
the order got changed while resolving merge conflicts. reverted to original order.